src/pint/models/timing_model.py

```diff
@@ -1736,55 +1783,63 @@ def designmatrix(self, toas, acc_delay=None, incfrozen=False, incoffset=True):
         ----------
         toas : pint.toa.TOAs
             The TOAs at which to compute the design matrix.
-        acc_delay
-            ???
         incfrozen : bool
             Whether to include frozen parameters in the design matrix
         incoffset : bool
             Whether to include the constant offset in the design matrix
+        calctype : {'modelF0', 'numerical', 'taylor'}
```

Comment:
Is there a way to make the possibilities for `calctype` the same here as in other functions that accept this parameter? I guess it's a bit trivial for `get_spin_freq` but it's nicer to have a more uniform calling convention.

Reply:
They are all identical now (including with the docstrings), except that `.designmatrix` also allows a `phase` option.